modules/irsa/main.tf

```diff
@@ -2,10 +2,6 @@ resource "kubernetes_namespace_v1" "irsa" {
   count = var.create_kubernetes_namespace && var.kubernetes_namespace != "kube-system" ? 1 : 0
   metadata {
     name = var.kubernetes_namespace
-
-    labels = {
-      "app.kubernetes.io/managed-by" = "terraform-aws-eks-blueprints"
```

Comment:
Can be renamed the key with `terraform.io/module`. This will be helpful for customers to identify the namespaces created by the Terraform module

Reply:
The same comment applies to all the below labels for Kuberentes Resources.